src/renderer/components/ft-input/ft-input.vue

```diff
@@ -49,6 +49,7 @@
         :placeholder="placeholder"
         :disabled="disabled"
         :spellcheck="spellcheck"
+        :aria-label="!showLabel ? placeholder : false"
```

Comment:
Vue 3 will turn `false` into `"false"` (so that we don't need the `String()` hacks for `aria-selected` anymore), so it's best to use `null` here so it still disappears in the future.

```suggestion
        :aria-label="!showLabel ? placeholder : null"
```